.eslintrc.json

```diff
@@ -7,7 +7,8 @@
     "no-invalid-this": "off",
     "@typescript-eslint/no-invalid-this": ["error"],
     "import/extensions": ["error", "always"],
-    "github/no-inner-html": "off"
+    "github/no-inner-html": "off",
+    "@typescript-eslint/no-unused-vars": ["error", { "vars": "all", "argsIgnorePattern": "^_" }]
```

Comment:
I needed to add this because in the dummy Trusted Types policies below, I'm throwing away the HTML that we receive in the `createHtml` call.

Reply:
Is it possible to add an ignore above the particular function?

Reply:
No, ESLint has a rule that prohibits ESLint directives by default: https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/no-use.html.

Considered disabling that rule, but that seems more intrusive than doing this.